app/mailers/admin_mailer.rb

```diff
@@ -13,6 +13,12 @@ def send_public_servant_account(admin:)
     mail to: admin.email, subject: I18n.t('mailer.subject.send_account_invitation', app_name: @app_name)
   end
 
+    def send_comptroller_account(admin:)
+    @admin = admin
+    @app_name = ENV["APP_NAME"]
+    mail to: admin.email, subject: I18n.t('mailer.subject.send_account_invitation', app_name: @app_name)
```

Comment:
Line is too long. [104/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.